thinkhazard/static/less/common.less

```diff
@@ -37,6 +37,13 @@ ul.horizontal li {
 .navbar {
   border: 0;
   .box-shadow(0px 2px 10px -1px rgba(0, 0, 0, 0.33));
+  li:last-child a{
+      padding-top: 0px;
+      padding-bottom: 0px;
+      margin-top: 29px;
+      margin-bottom: 29px;
+      border-left: 1px solid #d9d9d9;
```

Comment:
It a bit ugly when on click on the button.
![screenshot from 2015-07-02 15 05 25](https://cloud.githubusercontent.com/assets/319774/8477633/e6790b12-20cb-11e5-97dc-80aee992fe68.png)


It would be better to have a separate element for the divider than adding a border.

See https://www.google.fr/search?q=bootstrap+navbar+separator&oq=bootstrap+navbar+separa&aqs=chrome.1.69i57j69i59.9931j0j4&sourceid=chrome&es_sm=93&ie=UTF-8#q=bootstrap+navbar+divider&tbs=qdr:y
http://solvedstack.com/questions/vertical-divider-don-t-work-in-bootstrap-3